### ISSUE TITLE
Live-live replication doesn't work for the pcapi

### ISSUE THREAD

**Issue by gmh04** [edina/fieldtrip-gb#19]
The pcapi uses a token for client authentication that is currently stored in an sqlite database.  In live-live replication URLs don't always resolve because the token is stored on a single machine.  There are a number of solutions:

1)  Use vorlichdb as it is ahared by AT and KB.
2)  This could be unsavoury to some, therefore we could investigate clustering?
3)  Ben discussed a solution with Matt where something is added to the URL that identifies with machine the load balancer should go to. (what happens when the machine is down?)

**Comment by benismobile** [comment]
Ruben has suggested redis insteand of SQLLite

**Comment by gmh04** [comment]
Much discussion in the past about dropbox tokens expiring, check this out:

https://www.dropbox.com/developers/support#token-expiration

to sum up: "App access tokens effectively don't expire"

**Comment by benismobile** [comment]
think George is now looking at replicated Postgres solution

**Comment by gmh04** [comment]
Yes, as discussed, the easiest and best understood way to achieve this is to use vorlichdb.

**Comment by gmh04** [comment]
rainbow is now using a local postgres database and ready to be tested.  Tokens migrated from sqlite.

live will use vorlichdb
